packages/text-field/README.md

```diff
@@ -62,6 +62,7 @@ dense | Boolean | Enables dense variant.
 floatingLabelClassName | String | An optional class added to the floating label element fo the text field.
 fullWidth | Boolean | Enables fullWidth variant.
 helperText | Element | Helper text that appears below the text field. Use the `<HelperText>` component that comes with this package.
+isRtl | Boolean | Updates the `<NotchedOutline>` to the correct orientation (left or right).
```

Comment:
This description is an implementation detail, maybe say instead "Whether the direction of the text field element is set to RTL."